### DISCUSSION TITLE
Immich android offline usage

### DISCUSSION THREAD

**Discussion by martadinata666** [immich-app/immich#1979] [Q&A] [answered]
I had deployed immich and matched version between the android app and the web, but somehow the android app keep disconnecting due android no internet status. It works a few seconds, then is stuck, and returns to the login interface. Is this intended behavior? or I'm missing something? thanks

**Comment by Allram** [comment, marked as answer]
Offline mode is not implemented yet :⁠-⁠)

**Reply by EinToni** [reply]
Is offline app usage planned to be implemented?

Edit: Offline usage is already available, but only when the internet connection of the phone is really off. I have my immich server only available in my local network, when leaving that network and then opening the app with mobile internet connection turned on, the app wont open. When then turning the internet off the app opens :)

**Reply by tulku** [reply]
Should it work now? If I try opening the app without connection to the server it asks me to login again
